qcodes_contrib_drivers/drivers/Bilt/ITest.py

```diff
@@ -0,0 +1,469 @@
+# This Python file uses the following encoding: utf-8
+# Loick Le Guevel, 2019
+# Etienne Dumur <etienne.dumur@gmail.com>, 2021
+
+from typing import Union, Tuple
+from functools import partial
+from math import ceil
+from time import sleep
+
+from qcodes.instrument.base import Instrument
+from qcodes.instrument.channel import InstrumentChannel, ChannelList
+from qcodes.instrument.channel import MultiChannelInstrumentParameter
+from qcodes.instrument.visa import VisaInstrument
+
+
+class iTestChannel(InstrumentChannel):
+    """
+    A single channel of iTest.
+    """
+    
+    def __init__(self, parent: Instrument,
+                       name: str,
+                       chan_num: int) -> None:
+        """
+        Args:
+            parent: The instrument to which the channel is attached.
+            name: The name of the channel.
+            chan_num: The number of the channel in question
+        """
+        super().__init__(parent, name)
+        
+        self.chan_num = chan_num
+        # Get channel id
+        i, c = round(chan_num/4.)+1, chan_num%4
+        self.chan_id = 'i{};c{};'.format(i,c)
+        
+        self.add_parameter('v',
+                           label='Channel {} voltage'.format(chan_num),
+                           unit='V',
+                           docstring='Voltage of the channel in volt.',
+                           get_cmd=partial(self._parent._get_voltage, chan_num),
+                           set_cmd=partial(self._parent._set_voltage, chan_num),
+                           get_parser=float
```

Comment:
Doesn't it need any validator for set values?
It can prevent setting out of limit values in software level.